CWE/events/italy.txt

```diff
@@ -825,31 +464,6 @@ country_event = {
 
 }
 
-#Pavarotti dies
```

Comment:
Please do not remove content that already works unless there is a replacement event for it or the even is causing issues.

Reply:
I prefered to re-purpose the event as the Pavarotti death seems kinda unfitting with the whole mod as no nation has events for pop culture deaths and ha sno relevance to gameplay nor any effect

Reply:
Okay